sorc/_workaround_/MPAS-Model/Registry.xml

```diff
@@ -1027,6 +1027,7 @@
 			<var name="isltyp"/>
 			<var name="lai"/>
 			<var name="sfc_albedo"/>
+                        <var name="dzs"/>
```

Comment:
It looks like you used spaces for the leading whitespace in line 1030
Other lines, such as line 1029, use 3 tabs.

I suggest we copy line 1029 to line 1030 and then change "sfc_albedo" to "dzs"